src/pyggdrasil/tree_inference/_interface.py

```diff
@@ -50,3 +50,7 @@
 # i.e. [0,0,...0,0,1]
 # A node is its own ancestor.
 AncestorMatrix = Array
+
+# Observational Error rates
+# tuple of (alpha, beta)
+ErrorRates = Tuple[float, float]
```

Comment:
In the future (when e.g., Euler's Python is sufficiently modern. Or perhaps it's already?), you won't need `Tuple` and ordinary `tuple[float, float]` will work (similarly for `list` and `dict`).